apps/src/aiEvaluation/aiEvaluationApi.ts

```diff
@@ -17,9 +17,14 @@ export interface AIResponse {
   aiReasoning: string;
   evaluationCriteria: string;
   skillEvaluations?: [AIResponse];
+  id?: number;
 }
 
-export interface StudentWorkEvaluation extends StudentAnswer, AIResponse {}
+export interface StudentWorkEvaluation extends StudentAnswer, AIResponse {
```

Comment:
Should this be a `type` instead of an `interface` - I don't see us extending it...

It is also possible this shouldn't extend `StudentAnswer`... looking into that now.

Reply:
🤔 The StudentWorkEvaluation includes all of the fields in a StudentAnswer + all of the fields in an AIResponse, right?

Reply:
Yeah, I was worried I was cramming too much into one "type" - like perhaps we should keep them separate and pass in a StudentWorkEvaluation and a separate AiResponse... but I think the AiReponse IS part of the evaluation... so I will keep it together until I bump into something that makes me feel differently.